genomic_data_service/regulome_atlas.py

```diff
@@ -16,7 +16,7 @@
 
 try:
     TRAINED_REG_MODEL = pickle.load(
-        open('./ml_models/rf_model.sav', 'rb')
+        open('./ml_models/rf_model1.0.1.sav', 'rb')
```

Comment:
We should also update the model in s3, as well as the code that fetches the models.

Reply:
I already uploaded the model to s3 bucket and changed the code to fetch that model.